src/pages/MenuPage.js

```diff
@@ -235,8 +239,13 @@ const mapDispatchToProps = dispatch => ({
 const mapStateToProps = state => {
   const { modules, user } = state;
   const { currentUser } = user;
-  const { usingDispensary } = modules;
-  return { usingDispensary, usingModules: usingDispensary, isAdmin: currentUser?.isAdmin };
+  const { usingDashboard, usingDispensary } = modules;
```

Comment:
you can destructure `usingModules` here as it is being set in state in the `ModulesReducer` (line 26 in modulesReducer.js), then you don't have to do the check in line 247

In fact, you could just do a `return {...modules, isAdmin: currentUSer?.isAdmin }`  - but I kind of like taking the values out .. it is more explicit and helpful for understanding. What do you think?

Reply:
Yes, I agree! This is a great way to reduce code and keep it clear at the same time.
I would rather go with the one you mentioned but keeping the destructure for modules in this page. 
Good to know we have the chance to just update `const modules = { usingDashboard, usingDispensary, usingVaccines };` in `ModulesReducer` and we don't need to make modifications here:
```
const mapStateToProps = state => {
  const { modules, user } = state;
  const { currentUser } = user;
  return { ...modules, isAdmin: currentUser?.isAdmin };
};
```
Anyway, I agree that it would be good to have ` { usingDashboard, usingDispensary, usingVaccine } = modules` in this page to improve understanding. Otherwise let me know and we took those lines out.
Thanks btw :-)

Reply:
I agree.. explicit is best :)